data/templates/install-coreos.sh

```diff
@@ -3,5 +3,11 @@ set -e
 CLOUD_CONFIG_FILE=pxe-cloud-config.yml
 curl -o $CLOUD_CONFIG_FILE http://<%=server%>:<%=port%>/api/current/templates/$CLOUD_CONFIG_FILE?nodeId=<%=nodeId%>
 sudo coreos-install -d <%=installDisk%> -c $CLOUD_CONFIG_FILE -b <%=repo%>
+
+# Customizations for supporting CoreOS Ignition:
+mkdir /mnt/coreos
+mount <%=installDisk%>6 /mnt/coreos/
```

Comment:
@tgelter so it looks like either an annotation or a variable whose name could hint what '6' means is needed. could be OEM_PARTITION or any meaningful name you think.